### PR TITLE
Feature/layer UI improvements

### PR DESCRIPTION
**Pull request by TomSimons**
- Added scroll view to layer panel
- Fixed layers not reparenting to closed layer group
- reduced scroll view elasticity
- Removed captions in settings panel according to latest design

### CODE REVIEW

File: Assets/Scripts/UI/EqualSpacingCalculator.cs [hunk 1]
```diff
@@ -37,6 +38,8 @@ private void Start()
 
         void CalculateSpacing()
         {
+            // rt.SetRect(0, 0, 16, 16);
```
Comment (mvriel) [inline Assets/Scripts/UI/EqualSpacingCalculator.cs:41]: Dit kan weg

File: Assets/Scripts/UI/EqualSpacingCalculator.cs [hunk 2]
```diff
@@ -1,6 +1,7 @@
 using System;
 using System.Collections;
 using System.Collections.Generic;
+using SLIDDES.UI;
```
Comment (mvriel) [inline Assets/Scripts/UI/EqualSpacingCalculator.cs:4]: Dit kan weg

File: Assets/Scripts/UI/AdvancedScrollView.cs [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            if (gameObject.activeInHierarchy)
+                StartCoroutine(ResetScrollActiveAtEndOfFrame());
+        }
+
+        private IEnumerator ResetScrollActiveAtEndOfFrame()
+        {
+            yield return new WaitForEndOfFrame(); //wait for the end of the frame, because the layout needs to be updated for this calculation to work
+            var fitsInViewport = scrollRect.viewport.rect.height < scrollRect.content.rect.height;
+            scrollRect.movementType = fitsInViewport ? ScrollRect.MovementType.Elastic : ScrollRect.MovementType.Clamped; //scrollRect.verticalScrollbar.gameObject.activeInHierarchy;
+        }
```
Comment (sambaas) [inline Assets/Scripts/UI/AdvancedScrollView.cs:39]: comment kan weg

**Review by sambaas — APPROVED** [review]
ship it. kan alleen nog het eea. aan oude commented code weg

**Pull request merged**